### ISSUE TITLE
RTL languages support

### ISSUE THREAD

**Issue by themedleb** [codimd/server#458]
Support for auto language directions (include `dir=“auto”` in the HTML markup of each element like paragraph, heading, list, to-do list, …).
This is so much important for RTL languages like Arabic, Hebrew and others.

**Comment by DerMolly** [comment]
The [new frontend](https://github.com/codimd/react-client) was built with this in mind. And I hope we added it in all the important places already. Feel free to try out the current WIP react-client and tell us if you find anything that's missing.

**Comment by themedleb** [comment]
Oh I didn't know it was already there in the coming version!

I successfully installed the react client but it didn't open the port 3000 for me, I checked the opened ports in my computer using this command `sudo netstat -tulpn | grep LISTEN` and 3000 is not there.

Is there anything else I should do?

Thank you.

**Comment by DerMolly** [comment]
With installed i presume you mean on your local machine (as of the now the react-client can't be used with any backend).
After you executed `yarn install` and `yarn start` your browser should show you the new frontend or there should an error message in the terminal…

**Comment by themedleb** [comment]
I did `"yarn start"` and it finally worked!! 

I like the new UI, it looks more modern and I guess this is just the beginning.

For the RTL languages support, I noticed the attribute `dir="auto"` in the `id="root"` and also in the class of the banner `"fade mb-0 text-center d-flex flex-row justify-content-center alert alert-primary show"`, but unfortunately it is not affecting the content (both the editing and viewing area).

**Comment by themedleb** [comment]
I think I found the solution:

The problem: the body has `text-align: left;`, this is what prevents direction to change even if we include `dir="auto"` everywhere

The solution:

1. Change it from "left" to "inherit".
But this alone isn't enough, that's why we need to do the next 2 steps.

2. Add `dir="auto"` to the element that has the class `class="markdown-body flex-fill d-flex flex-column align-items-center"`, this is the class that contains the content of the View mode.

3. Add `dir="auto"` to the element that has the class `class="CodeMirror-code"`, this is the class that contains the content of the Edit mode.

It worked this way in both View and Edit mode.
I hope this helps.

**Comment by mrdrogdrog** [comment]
Yes, that is indeed helpful! But please open an issue in the react-client repo.
